zerver/tests/test_decorators.py

```diff
@@ -156,13 +164,14 @@ class Request(object):
         self.assertEqual(result, 21)
 
     def test_REQ_argument_type(self):
-
+        # type: () -> None
         @has_request_variables
         def get_payload(request, payload=REQ(argument_type='body')):
+            # type: (HttpRequest, Dict[str, Dict]) -> Dict[str, Dict]
             return payload
 
         class MockRequest(object):
-            body = {}
+            body = {} # type: Any
```

Comment:
Please check the lines after that, when request is assigned the class mockrequest, we use request.body and assign it a string. That's the reason I've used Any